modules/renter/hostdb/scan.go

```diff
@@ -99,7 +99,7 @@ func (hdb *HostDB) decrementReliability(addr modules.NetAddress, penalty types.C
 	// Look up the entry and decrement the reliability.
 	entry, exists := hdb.allHosts[addr]
 	if !exists {
-		// TODO: should panic here
+		build.Critical("host to be decremented did not exist in hostdb")
```

Comment:
This statement is a TODO because there's a bug where the code is getting hit frequently I believe. We had a hard time finding it, and so chose to leave the bug rather than to build.Critical, otherwise our daemons would be crashing all the time.

Basically, need to fix the bugs before we can merge this.